analyzers/tests/SonarAnalyzer.Test/TestCases/ExceptionsShouldBeLogged.Microsoft.Extensions.Logging.cs

```diff
@@ -0,0 +1,405 @@
+﻿using System;
+using Microsoft.Extensions.Logging;
+using Microsoft.Extensions.Logging.Abstractions;
+
+// Logging methods from: https://learn.microsoft.com/en-us/dotnet/api/microsoft.extensions.logging.loggerextensions?view=dotnet-plat-ext-8.0
```

Comment:
```suggestion
// Logging methods from: https://learn.microsoft.com/en-us/dotnet/api/microsoft.extensions.logging.loggerextensions
```

Reply:
I prefer to keep this, do document the exact version we support. Later, it will be easier to compare the supported overloads if we know the version.